styles/pages/settings/settings.scss

```diff
@@ -260,7 +261,7 @@ $page-width: 1920px - $rightnav-width;
 	&__subitem-label {
 		padding-left: 24px;
 
-		@include mixin.font-styles($use-header: true, $light-header: true);
```

Comment:
There are instances like these where both were true passed in, but the separation of the mixins prevents `font-family` from ever being the "light" header version. Is this intentional?